During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/odoo17/odoo/addons/base/models/ir_cron.py", line 138, in _process_jobs
    registry = odoo.registry(db_name)
  File "/opt/odoo17/odoo/__init__.py", line 125, in registry
    return modules.registry.Registry(database_name)
  File "/opt/odoo17/odoo/modules/registry.py", line 89, in __new__
    return cls.new(db_name)
  File "<decorator-gen-16>", line 2, in new
  File "/opt/odoo17/odoo/tools/func.py", line 87, in locked
    return func(inst, *args, **kwargs)
  File "/opt/odoo17/odoo/modules/registry.py", line 113, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "/opt/odoo17/odoo/modules/loading.py", line 476, in load_modules
    processed_modules += load_marked_modules(env, graph,
  File "/opt/odoo17/odoo/modules/loading.py", line 362, in load_marked_modules
    graph.add_modules(env.cr, module_list, force)
  File "/opt/odoo17/odoo/modules/graph.py", line 71, in add_modules
    info = odoo.modules.module.get_manifest(module)
  File "/opt/odoo17/odoo/modules/module.py", line 369, in get_manifest
    return copy.deepcopy(_get_manifest_cached(module, mod_path))
  File "/opt/odoo17/odoo/modules/module.py", line 373, in _get_manifest_cached
    return load_manifest(module, mod_path)
  File "/opt/odoo17/odoo/modules/module.py", line 319, in load_manifest
    manifest.update(ast.literal_eval(f.read()))
  File "/opt/odoo17/odoo/tools/_monkeypatches.py", line 73, in literal_eval
    return orig_literal_eval(expr)
  File "/usr/lib/python3.10/ast.py", line 64, in literal_eval
    node_or_string = parse(node_or_string.lstrip(" \t"), mode='eval')
  File "/usr/lib/python3.10/ast.py", line 50, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 43
    'ifdw_social_media_monitoring/static/src/js/analytics_views.js',,
                                                                    ^
SyntaxError: invalid syntax
